admin/base/urls.py

```diff
@@ -10,7 +12,9 @@
     url(base_pattern,
         include(patterns('',
                          url(r'^$', views.root),
+                         url(r'^django_admin/', include(admin.site.urls)),
```

Comment:
@Ghalko I'm sure we could. Are we thinking of obfuscating by url name, while keeping intelligible namespaces for use in code?